src/main/webapp/app/complaints/complaints-for-tutor/complaints-for-tutor.component.ts

```diff
@@ -91,15 +91,15 @@ export class ComplaintsForTutorComponent implements OnInit {
     }
 
     private refreshLock() {
-        this.complaintResponse = this.complaint.complaintResponse!;
+        this.complaintResponse = this.complaint?.complaintResponse!;
```

Comment:
does it make sense to use the `?` and then the `!` operator?

Reply:
no thats true it doesnt make sense, i overlooked it the "!"

Reply:
oh actually it does make sense, since the complaint is optional in this case and the complaintResponse is also optional.
We could also check if both exist before assigning the values, does that make more sense?